Assets/Scripts/Audio/FmodEventPoolableObject.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using UnityEngine;
+
+public class FmodEventPoolableObject : MonoBehaviour
```

Comment:
Again, doesn't necessarily need to be a MonoBehaviour

Reply:
I'll look into this some more. I made this class a MonoBehaviour so I could define the fmod event data through prefabs, though your suggestion of using scriptable objects might be the way to go.

Reply:
Restructured FmodEventPool to automatically generate pools of FmodEventPoolableObjects from FmodEventPoolableData scriptable objects in the resources folder. None of these scripts use Monobehavior. FmodFacade now holds an FmodEventPool object for pooling operations.